splash-screen-hanging.md

```diff
@@ -0,0 +1,15 @@
+# Stuck on the Splash Screen?
+
+If you've recently encountered this popup while developing your app, chances are something has gone wrong with the SplashScreen component. Here are a couple of possible reasons why you might be seeing this with your app: 
```

Comment:
```suggestion
It can be frustrating to work on your app when you can't even get it to load past the splash screen. Here are a couple of possible reasons why you might be seeing this with your app:
```